src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file
+ * @brief test for pthread_kill
+ *
+ * @author  XU Chenglin
+ * @date    09/03/2020
+ */
+
+#include <pthread.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <signal.h>
+#include <embox/test.h>
+
+
```

Comment:
rm empty lines

Reply:
Sorry for messing up the code style. I will modify it carefully.